lib/plenario/changesets/meta_changesets.ex

```diff
@@ -164,17 +164,18 @@ defmodule Plenario.Changesets.MetaChangesets do
   defp validate_refresh_ends_on(changeset), do: changeset
 
   defp set_slug(%Ecto.Changeset{valid?: true, changes: %{name: name}} = changeset) do
-    slug = Slug.slugify(name)
+    ignore =
+      Regex.scan(~r/[^\x00-\x7F]/, name)
+      |> List.to_string()
```

Comment:
Non-ASCII characters. 

Say we have a data set `東京キャブ乗車2017` (Tokyo Cab Rides 2017). Slugify would first translate the Kanji to Chinese (wrong, given the context) and then transliterate all of that to "English". Their result would be `dongjingkiyabuchengche2017` instead of the (slightly more accurate) `tōkyō-kyabu-jōsha-2017`.

What this does is says _screw it, this is obviously specific enough to a demographic that I shouldn't try to Anglicize it_ and goes on its merry way. Basically, I'm punting on having to decide who gets translated and how accurately.

Reply:
So to clarify, it's leaving the kanji in place in the slug? And nothing in our pipeline chokes on that?

Reply:
Correct. Everything else should be just fine.